library/connect.c

```diff
@@ -95,6 +98,7 @@ void on_write_completed(struct ziti_conn *conn, struct ziti_write_req_s *req, in
 
         if (status < 0) {
             conn->state = Closed;
+            ZITI_LOG(TRACE, "conn %p state is now Closed", conn);
```

Comment:
I'd prefer to add `conn_id` here and other places where we log with `%p`

Reply:
done